<!DOCTYPE html><html><head><meta charSet="utf-8"/><meta name="viewport" content="width=device-width"/><link rel="icon" href="/mapp-dws-public/favicon.ico"/><title>Pricelist</title><meta name="robots" content="index,follow"/><meta name="googlebot" content="index,follow"/><meta name="description" content="Personal knowledge space"/><meta property="og:title" content="Pricelist"/><meta property="og:description" content="Personal knowledge space"/><meta property="og:url" content="https://mapp-metabolomics-unit.github.io/mapp-dws-public/notes/RBxrtKZIWsMnHMVAyLlmc/"/><meta property="og:type" content="article"/><meta property="article:published_time" content="5/7/2021"/><meta property="article:modified_time" content="6/8/2021"/><link rel="canonical" href="https://mapp-metabolomics-unit.github.io/mapp-dws-public/notes/RBxrtKZIWsMnHMVAyLlmc/"/><meta name="next-head-count" content="14"/><link rel="preload" href="/mapp-dws-public/_next/static/css/8c1cf0f05ac8a12a.css" as="style"/><link rel="stylesheet" href="/mapp-dws-public/_next/static/css/8c1cf0f05ac8a12a.css" data-n-g=""/><noscript data-n-css=""></noscript><script defer="" nomodule="" src="/mapp-dws-public/_next/static/chunks/polyfills-c67a75d1b6f99dc8.js"></script><script src="/mapp-dws-public/_next/static/chunks/webpack-7f33832883bb3344.js" defer=""></script><script src="/mapp-dws-public/_next/static/chunks/framework-28c999baf2863c3d.js" defer=""></script><script src="/mapp-dws-public/_next/static/chunks/main-e6b7bc32869bfae1.js" defer=""></script><script src="/mapp-dws-public/_next/static/chunks/pages/_app-541f8555c61e5774.js" defer=""></script><script src="/mapp-dws-public/_next/static/chunks/935-4dee79e80b8641c6.js" defer=""></script><script src="/mapp-dws-public/_next/static/chunks/6-50972def09142ee2.js" defer=""></script><script src="/mapp-dws-public/_next/static/chunks/pages/notes/%5Bid%5D-78d472fa3b924116.js" defer=""></script><script src="/mapp-dws-public/_next/static/HcHSndH05GsIFs4lOyTZF/_buildManifest.js" defer=""></script><script src="/mapp-dws-public/_next/static/HcHSndH05GsIFs4lOyTZF/_ssgManifest.js" defer=""></script></head><body><div id="__next" data-reactroot=""><section class="ant-layout" style="width:100%;min-height:100%"><header class="ant-layout-header" style="position:fixed;isolation:isolate;z-index:1;width:100%;border-bottom:1px solid #d4dadf;height:64px;padding:0 24px 0 2px"><div class="ant-row ant-row-center" style="max-width:992px;justify-content:space-between;margin:0 auto"><div style="display:flex" class="ant-col ant-col-xs-20 ant-col-sm-4"></div><div class="ant-col gutter-row ant-col-xs-0 ant-col-sm-20 ant-col-md-20 ant-col-lg-19"><div class="ant-select ant-select-lg ant-select-auto-complete ant-select-single ant-select-allow-clear ant-select-show-search" style="width:100%"><div class="ant-select-selector"><span class="ant-select-selection-search"><input type="search" autoComplete="off" class="ant-select-selection-search-input" role="combobox" aria-haspopup="listbox" aria-owns="undefined_list" aria-autocomplete="list" aria-controls="undefined_list" aria-activedescendant="undefined_list_0" value=""/></span><span class="ant-select-selection-placeholder">For full text search please use the &#x27;?&#x27; prefix. e.g. ? Onboarding</span></div></div></div><div style="display:none;align-items:center;justify-content:center" class="ant-col ant-col-xs-4 ant-col-sm-4 ant-col-md-0 ant-col-lg-0"><span role="img" aria-label="menu" style="font-size:24px" tabindex="-1" class="anticon anticon-menu"><svg viewBox="64 64 896 896" focusable="false" data-icon="menu" width="1em" height="1em" fill="currentColor" aria-hidden="true"><path d="M904 160H120c-4.4 0-8 3.6-8 8v64c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-64c0-4.4-3.6-8-8-8zm0 624H120c-4.4 0-8 3.6-8 8v64c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-64c0-4.4-3.6-8-8-8zm0-312H120c-4.4 0-8 3.6-8 8v64c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-64c0-4.4-3.6-8-8-8z"></path></svg></span></div></div></header><section class="ant-layout" style="margin-top:64px;display:flex;flex-direction:row"><div class="site-layout-sidebar" style="flex:0 0 auto;width:calc(max((100% - 992px) / 2, 0px) + 200px);min-width:200px;padding-left:calc((100% - 992px) / 2)"><aside class="ant-layout-sider ant-layout-sider-dark" style="position:fixed;overflow:auto;height:calc(100vh - 64px);background-color:transparent;flex:0 0 200px;max-width:200px;min-width:200px;width:200px"><div class="ant-layout-sider-children"></div></aside></div><main class="ant-layout-content side-layout-main" style="max-width:1200px;min-width:0;display:block"><div style="padding:0 24px"><div class="main-content" role="main"><div class="ant-row"><div class="ant-col ant-col-24"><div class="ant-row" style="margin-left:-10px;margin-right:-10px"><div style="padding-left:10px;padding-right:10px" class="ant-col ant-col-xs-24 ant-col-md-18"><div><h1 id="pricelist">Pricelist<a aria-hidden="true" class="anchor-heading icon-link" href="#pricelist"></a></h1>
<p>A new price list and format needs to be defined</p>
<p>Fisrt I would like to compare what other labs in CH and abroad are asking for </p>
<h1 id="bern-proteomics-mass-spectrometry-core-facility">Bern Proteomics Mass Spectrometry Core Facility<a aria-hidden="true" class="anchor-heading icon-link" href="#bern-proteomics-mass-spectrometry-core-facility"></a></h1>
<p><a href="https://www.pmscf.dbmr.unibe.ch/services/fees/">https://www.pmscf.dbmr.unibe.ch/services/fees/</a></p>
<h1 id="functional-genomics-center-zurich">Functional Genomics Center Zurich<a aria-hidden="true" class="anchor-heading icon-link" href="#functional-genomics-center-zurich"></a></h1>
<p><a href="https://fgcz.ch">https://fgcz.ch</a>
<a href="https://fgcz-bfabric.uzh.ch/wiki/metabolomics.pricelist">https://fgcz-bfabric.uzh.ch/wiki/metabolomics.pricelist</a></p>
<p>LCMS / sample 100 CHF (internal) (incl. data acquisition, data storage, data processing; a min. of 12 samples is expected)
LCMS / sample 200 CHF (external)</p>
<p>Metabolomics User lab Access 350 /day (internal)
Metabolomics User lab Access 1000 /  day (external)</p>
<h1 id="biotech-cornell">Biotech Cornell<a aria-hidden="true" class="anchor-heading icon-link" href="#biotech-cornell"></a></h1>
<p><a href="https://www.biotech.cornell.edu/core-facilities-brc/price-list/30">https://www.biotech.cornell.edu/core-facilities-brc/price-list/30</a></p>
<p>Description	Internal price (Cornell and Cornell affiliates)	External price
Metabolomics (per sample)	$5	$8
Lipidomics (per sample)	$8	$13
GCMS derivitization (per sample)	$12	$19</p>
<p>LC / MS Sample Analysis
Description	Internal price (Cornell and Cornell affiliates)	External price
Unargeted metabolomics analysis setup	$100	$160
Untargeted metabolomics RP analysis (per hour)	$100	$160
Untargeted metabolomics HILIC analysis (per hour)	$100	$160
Untargeted lipidomics analysis setup	$150	$240
Untargeted lipidomics analysis (per hour)	$100	$160
Creatinine assay setup	$100	$160
Creatinine assay	$10/sample	$16/sample</p>
<p>GC / MS Sample Analysis
Description	Internal price (Cornell and Cornell affiliates)	External price
Untargeted metabolomics analysis setup	$150	$240
Untargeted metabolomics analysis (per hour)	$100	$160</p>
<p>LC / MS Data Interpretation
Description	Internal price (Cornell and Cornell affiliates)	External price
Basic untargeted metabolomics with Compound Discoverer (one-time)	$200	$320
Basic untargeted lipidomics with Lipidsearch (one-time)	$300	$480
Advanced Analysis/Special Requirements (per hour)	$100	$160
GC / MS Data Interpretation
Description	Internal price (Cornell and Cornell affiliates)	External price
Basic Non-Targeted Metabolomics with TraceFinder, Compound Discoverer (one-time)	$400	$640
Advanced Analysis/Special Requirements (per hour)	$100	$160</p>
<h1 id="uc-davis-west-coast-metabo-center">UC Davis West Coast Metabo center<a aria-hidden="true" class="anchor-heading icon-link" href="#uc-davis-west-coast-metabo-center"></a></h1>
<p><a href="https://metabolomics.ucdavis.edu/core-services/rates-and-costs-for-services">https://metabolomics.ucdavis.edu/core-services/rates-and-costs-for-services</a></p>
<h1 id="uthsc">UTHSC<a aria-hidden="true" class="anchor-heading icon-link" href="#uthsc"></a></h1>
<p><a href="https://www.uthsc.edu/research/institutional-cores/pmc/documents/fy20-pmc-mpms-price-list.pdf">https://www.uthsc.edu/research/institutional-cores/pmc/documents/fy20-pmc-mpms-price-list.pdf</a></p>
<p>Targeted metabolite analyses:
$107 / sample   </p>
<p>Identification of Unknown Metabolites
~ $ 5,000 / project  </p>
<h1 id="duke">Duke<a aria-hidden="true" class="anchor-heading icon-link" href="#duke"></a></h1>
<p><a href="https://genome.duke.edu/cores-and-services/proteomics-and-metabolomics/pricing">https://genome.duke.edu/cores-and-services/proteomics-and-metabolomics/pricing</a></p>
<p>Sample Cleanup Procedures (cost per method)	$48	$77
Solid Phase Extraction (C18, HLB, MCX)		</p>
<p>Quantitative Analysis (Metabolomics)</p>
<p>All prices are on a per-sample basis.</p>
<p>Duke	Non-Duke
AbsoluteIDQ p180 Panel (kit from Biocrates, Inc.)<em>	$52	$84
Absolute IDQ p400 HR Panel (kit from Biocrates, Inc.)</em>	$72	$116
MxP Quant 500 Panel (kit from Biocrates, Inc.)<em>	$84	$135
Bile Acids Panel (kit from Biocrates, Inc.)</em>	$41	$66
Oxylipins/Eicosanoid Panel<strong>	$145	$233
Energy Metabolite Panel	$64	$103
Methionine Pathway Panel</strong>	$88	$142
Hydroxycholesterols Panel<strong>	$130	$209
27-Hydroxycholesterol quantitation</strong>	$76	$122
Fatty Acids Panel	$74	$119
Fast HR Metabolite Profiling (RPLC OR HILIC, ESI+ OR ESI-)	$142	$229
High Resolution Polar Metabolite Profiling, ESI+	$207	$333
High Resolution Lipid Profiling, ESI+ and ESI-	$252	$406
PK Analysis - method development	Contact	Contact
per sample with in-house developed method<strong>	$29	$47
Polyamine Panel</strong>	$57	$92
Short Chain Fatty Acids Analysis<strong>	$61	$98
Sphingosine S1P Panel</strong>	$44	$71
ZipChip Profiling Method	$66	$106</p>
<p>Data Analysis</p>
<p>Duke	Non-Duke
Custom Database Setup for MASCOT or Identify	$25	$40
Additional Database Searches	$32	$52
Consulting and Extensive Data Mining (per hour)	$64	$103</p>
<h1 id="metabo-centre-ca">Metabo centre ca<a aria-hidden="true" class="anchor-heading icon-link" href="#metabo-centre-ca"></a></h1>
<p><a href="https://www.metabolomicscentre.ca/services">https://www.metabolomicscentre.ca/services</a></p>
<p>Untargeted Metabolomics by RPLC-QTOF-MS and MS/MS</p>
<blockquote>
<p>5000 features
Academic/Government:</p>
</blockquote>
<p>$140 for 2 RPLC-MS runs per sample (relative quantification);
$280 for 2 RPLC-MS/MS runs per study (metabolite identification)
$75/hour for data analysis
Industry:</p>
<p>$336 per sample (relative quantification)
$672 per study (metabolite identification)
$180/hour for data analysis
20-200 µL or 10-50 mg lyophilized powder
Untargeted metabolomics by GC×GC-TOFMS (EI only). Sample introduction options include liquid, SPME, direct headspace, thermal desorption</p>
<blockquote>
<p>2000
Academic/Government:</p>
</blockquote>
<p>$200 + sample prep
$75/hour for data analysis
Industry:</p>
<p>-$400 + sample preparation assuming standard GC×GC column configuration works – i.e. no custom method development - $180/hour for data analysis</p>
<p>Untargeted metabolomics by UPLC-FT-MS
6000 - 10000
Academic/Government:</p>
<p>$400 (1st sample)
$120 for 2 UPLC-MS runs per sample
$240 for 4 UPLC-MS runs per sample
$75/hour for data analysis
Industry:</p>
<p>$800 (1st sample)
$240 for 2 UPLC-MS runs per sample
$480 for 4 UPLC-MS runs per sample
$150/hour for data analysis
75 µL</p>
<h1 id="colostate">Colostate<a aria-hidden="true" class="anchor-heading icon-link" href="#colostate"></a></h1>
<p><a href="https://www.research.colostate.edu/bio/metabolomics-pricing/">https://www.research.colostate.edu/bio/metabolomics-pricing/</a></p>
<p>CSU	Non-CSU Academic or Non-profit	Non-CSU Commercial
LC-Q-TOF-MS: Set-up Fee (charged regardless of sample #)	$147	$221	$368
LC-Q-TOF-MS: Instrument usage	$74/hour	$110/hour	$184/hour
LC-Q-TOF-MS: Sample Preparation	$74/hour	$110/hour	$184/hour
GC-MS: Set-up Fee (charged regardless of sample #)	$114	$172	$286
GC-MS Analysis – Instrument usage	$41/hour	$61/hour	$102/hour
GC-MS: Sample Preparation	$74/hour	$110/hour	$184/hour
GC-MS: Sample Derivatization	$13/sample	$19/sample	$32/sample
LC-QQQ-MS: Set-up Fee (charged regardless of sample #)	$155	$233	$389
LC-QQQ-MS: Instrument usage	$82/hour	$123/hour	$205/hour
LC-QQQ-MS: Assay development	$622	$932	$1,554
LC-QQQ-MS: Sample Preparation	$74/hour	$110/hour	$184/hour
General Data Analysis<em>	$74/hour	$110/hour	$184/hour
Data analysis or sample preparation training</em>	$134/hour	$202/hour	$336/hour
Non-targeted Metabolite Profiling data analysis – tier 1<strong>	$74/project	$110/project	$184/project
Non-targeted Metabolite Profiling data analysis – tier 2</strong>	$294/project	$441/project	$735/project
Non-targeted Metabolite Profiling data analysis – tier 3**	$1176/project	$1764/project	$2940/project
Pricing as of February 01, 2020. Prices are subject to change at any time.</p>
<ul>
<li>One hour minimum</li>
</ul>
<p>**Tier 1 (raw data files converted to .cdf format, sample key), Tier 2 (raw data files converted to cdf, XCMS based peak detection and alignment with output data in csv format, outlier detection, PCA, simple ANOVA p-values, figures of output) , Tier 3 (raw data files converted to cdf, XCMS based peak detection and alignment with output data in csv format, outlier detection, PCA, simple ANOVA p-values, figures of output, metabolite annotation).</p>
<p>Please note that costs are independent of result, and fees must still be charged for unsuccessful analysis in order to cover analysis time, instrument maintenance and consumables.</p>
<h1 id="griffin-group">Griffin group<a aria-hidden="true" class="anchor-heading icon-link" href="#griffin-group"></a></h1>
<p><a href="https://www.metabolomics.bioc.cam.ac.uk/Facility">https://www.metabolomics.bioc.cam.ac.uk/Facility</a></p>
<ol start="2">
<li>
<p>GC-FID analysis of total fatty acids. Fatty acids will be definitively identified by retention time matching with known standards and where necessary the acquisition of mass spectra using GC-MS and matching with compounds in the NIST library. Consumable cost: £24.00 per sample.</p>
</li>
<li>
<p>GC-MS analysis of the aqueous extract of tissues, cells or biofluids using a MSTFA derivatisation procedure as detailed in Atherton et al., 2006. Metabolites will be identified by mass spectral matching with the NIST library and other publically available resources e.g. the Fiehn lab database. Consumable cost: £27.00 per sample.</p>
</li>
<li>
<p>UPLC LC-MS analysis of lipids from tissue and serum extracts to identify bulk lipid changes as detailed in Roberts et al., 2008. This assay will use a UPLC QTOF Ultima and allows the analysis of intact lipids directly. Lipid identification will be carried out by MS/MS fragmentation and reference to libraries such as LipidMaps where possible. Consumable costs: £36.00 per sample.</p>
</li>
</ol>
<h1 id="mac-anderson">Mac Anderson<a aria-hidden="true" class="anchor-heading icon-link" href="#mac-anderson"></a></h1>
<p><a href="https://www.mdanderson.org/research/research-resources/core-facilities/metabolomics-facility/services.html">https://www.mdanderson.org/research/research-resources/core-facilities/metabolomics-facility/services.html</a></p>
<p>Non-targeted Metabolomics:
Profiling (complete)
Quantitative analysis of metabolites in non-targeted fashion from multiple fractions
Multiple configurations available
Pricing:
Internal &#x26; Academic (In Network)-$600/sample
Academic (Out of Network)-$960 (+60% Overhead)
Non-academic-$1200/sample (+60% Overhead)</p>
<h1 id="metabo-iigb">Metabo iigb<a aria-hidden="true" class="anchor-heading icon-link" href="#metabo-iigb"></a></h1>
<p><a href="https://metabolomics.iigb.ucr.edu/services-and-pricing/">https://metabolomics.iigb.ucr.edu/services-and-pricing/</a></p>
<p>Most services start at around $74/sample, but the cost is largely dependent on sample preparation complexity and sample set size. Please contact the Metabolomics Core Team (<a href="/mapp-dws-public/mailto:metabolomics@ucr.edu">metabolomics@ucr.edu</a>) to discuss options and pricing for your project.</p>
<hr>
<strong>Children</strong>
<ol>
<li><a href="/mapp-dws-public/notes/YSBdEmydk8XLo8ssSQec2">Eval</a></li>
<li><a href="/mapp-dws-public/notes/fUD4wPYZe4AE8Gslj4qoA">Ph</a></li>
<li><a href="/mapp-dws-public/notes/yiNaYRHMNL83IomXDrDSG">Shiny</a></li>
<li><a href="/mapp-dws-public/notes/AjS4TQTF1i254OvXs20UW">Table</a></li>
</ol></div></div><div style="padding-left:10px;padding-right:10px" class="ant-col ant-col-xs-0 ant-col-md-6"><div><div class=""><div class="ant-anchor-wrapper dendron-toc" style="max-height:calc(100vh - 64px);z-index:1"><div class="ant-anchor"><div class="ant-anchor-ink"><span class="ant-anchor-ink-ball"></span></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#bern-proteomics-mass-spectrometry-core-facility" title="Bern Proteomics Mass Spectrometry Core Facility">Bern Proteomics Mass Spectrometry Core Facility</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#functional-genomics-center-zurich" title="Functional Genomics Center Zurich">Functional Genomics Center Zurich</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#biotech-cornell" title="Biotech Cornell">Biotech Cornell</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#uc-davis-west-coast-metabo-center" title="UC Davis West Coast Metabo center">UC Davis West Coast Metabo center</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#uthsc" title="UTHSC">UTHSC</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#duke" title="Duke">Duke</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#metabo-centre-ca" title="Metabo centre ca">Metabo centre ca</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#colostate" title="Colostate">Colostate</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#griffin-group" title="Griffin group">Griffin group</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#mac-anderson" title="Mac Anderson">Mac Anderson</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#metabo-iigb" title="Metabo iigb">Metabo iigb</a></div></div></div></div></div></div></div></div></div></div></div><div class="ant-divider ant-divider-horizontal" role="separator"></div><footer class="ant-layout-footer" style="padding:0 24px 24px"></footer></main></section></section></div><script id="__NEXT_DATA__" type="application/json">{"props":{"pageProps":{"note":{"id":"RBxrtKZIWsMnHMVAyLlmc","title":"Pricelist","desc":"","updated":1623145533814,"created":1620404202286,"custom":{},"fname":"pricelist","type":"note","vault":{"fsPath":"vault"},"contentHash":"a279b68a765f1ec5a29cd7808046b795","links":[],"anchors":{"bern-proteomics-mass-spectrometry-core-facility":{"type":"header","text":"Bern Proteomics Mass Spectrometry Core Facility","value":"bern-proteomics-mass-spectrometry-core-facility","line":14,"column":0,"depth":1},"functional-genomics-center-zurich":{"type":"header","text":"Functional Genomics Center Zurich","value":"functional-genomics-center-zurich","line":18,"column":0,"depth":1},"biotech-cornell":{"type":"header","text":"Biotech Cornell","value":"biotech-cornell","line":29,"column":0,"depth":1},"uc-davis-west-coast-metabo-center":{"type":"header","text":"UC Davis West Coast Metabo center","value":"uc-davis-west-coast-metabo-center","line":66,"column":0,"depth":1},"uthsc":{"type":"header","text":"UTHSC","value":"uthsc","line":70,"column":0,"depth":1},"duke":{"type":"header","text":"Duke","value":"duke","line":81,"column":0,"depth":1},"metabo-centre-ca":{"type":"header","text":"Metabo centre ca","value":"metabo-centre-ca","line":125,"column":0,"depth":1},"colostate":{"type":"header","text":"Colostate","value":"colostate","line":170,"column":0,"depth":1},"griffin-group":{"type":"header","text":"Griffin group","value":"griffin-group","line":204,"column":0,"depth":1},"mac-anderson":{"type":"header","text":"Mac Anderson","value":"mac-anderson","line":216,"column":0,"depth":1},"metabo-iigb":{"type":"header","text":"Metabo iigb","value":"metabo-iigb","line":233,"column":0,"depth":1}},"children":["YSBdEmydk8XLo8ssSQec2","fUD4wPYZe4AE8Gslj4qoA","yiNaYRHMNL83IomXDrDSG","AjS4TQTF1i254OvXs20UW"],"parent":"nKP9hb3jEQKnhdgkHa5l0","data":{}},"body":"\u003ch1 id=\"pricelist\"\u003ePricelist\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#pricelist\"\u003e\u003c/a\u003e\u003c/h1\u003e\n\u003cp\u003eA new price list and format needs to be defined\u003c/p\u003e\n\u003cp\u003eFisrt I would like to compare what other labs in CH and abroad are asking for \u003c/p\u003e\n\u003ch1 id=\"bern-proteomics-mass-spectrometry-core-facility\"\u003eBern Proteomics Mass Spectrometry Core Facility\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#bern-proteomics-mass-spectrometry-core-facility\"\u003e\u003c/a\u003e\u003c/h1\u003e\n\u003cp\u003e\u003ca href=\"https://www.pmscf.dbmr.unibe.ch/services/fees/\"\u003ehttps://www.pmscf.dbmr.unibe.ch/services/fees/\u003c/a\u003e\u003c/p\u003e\n\u003ch1 id=\"functional-genomics-center-zurich\"\u003eFunctional Genomics Center Zurich\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#functional-genomics-center-zurich\"\u003e\u003c/a\u003e\u003c/h1\u003e\n\u003cp\u003e\u003ca href=\"https://fgcz.ch\"\u003ehttps://fgcz.ch\u003c/a\u003e\n\u003ca href=\"https://fgcz-bfabric.uzh.ch/wiki/metabolomics.pricelist\"\u003ehttps://fgcz-bfabric.uzh.ch/wiki/metabolomics.pricelist\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003eLCMS / sample 100 CHF (internal) (incl. data acquisition, data storage, data processing; a min. of 12 samples is expected)\nLCMS / sample 200 CHF (external)\u003c/p\u003e\n\u003cp\u003eMetabolomics User lab Access 350 /day (internal)\nMetabolomics User lab Access 1000 /  day (external)\u003c/p\u003e\n\u003ch1 id=\"biotech-cornell\"\u003eBiotech Cornell\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#biotech-cornell\"\u003e\u003c/a\u003e\u003c/h1\u003e\n\u003cp\u003e\u003ca href=\"https://www.biotech.cornell.edu/core-facilities-brc/price-list/30\"\u003ehttps://www.biotech.cornell.edu/core-facilities-brc/price-list/30\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003eDescription\tInternal price (Cornell and Cornell affiliates)\tExternal price\nMetabolomics (per sample)\t$5\t$8\nLipidomics (per sample)\t$8\t$13\nGCMS derivitization (per sample)\t$12\t$19\u003c/p\u003e\n\u003cp\u003eLC / MS Sample Analysis\nDescription\tInternal price (Cornell and Cornell affiliates)\tExternal price\nUnargeted metabolomics analysis setup\t$100\t$160\nUntargeted metabolomics RP analysis (per hour)\t$100\t$160\nUntargeted metabolomics HILIC analysis (per hour)\t$100\t$160\nUntargeted lipidomics analysis setup\t$150\t$240\nUntargeted lipidomics analysis (per hour)\t$100\t$160\nCreatinine assay setup\t$100\t$160\nCreatinine assay\t$10/sample\t$16/sample\u003c/p\u003e\n\u003cp\u003eGC / MS Sample Analysis\nDescription\tInternal price (Cornell and Cornell affiliates)\tExternal price\nUntargeted metabolomics analysis setup\t$150\t$240\nUntargeted metabolomics analysis (per hour)\t$100\t$160\u003c/p\u003e\n\u003cp\u003eLC / MS Data Interpretation\nDescription\tInternal price (Cornell and Cornell affiliates)\tExternal price\nBasic untargeted metabolomics with Compound Discoverer (one-time)\t$200\t$320\nBasic untargeted lipidomics with Lipidsearch (one-time)\t$300\t$480\nAdvanced Analysis/Special Requirements (per hour)\t$100\t$160\nGC / MS Data Interpretation\nDescription\tInternal price (Cornell and Cornell affiliates)\tExternal price\nBasic Non-Targeted Metabolomics with TraceFinder, Compound Discoverer (one-time)\t$400\t$640\nAdvanced Analysis/Special Requirements (per hour)\t$100\t$160\u003c/p\u003e\n\u003ch1 id=\"uc-davis-west-coast-metabo-center\"\u003eUC Davis West Coast Metabo center\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#uc-davis-west-coast-metabo-center\"\u003e\u003c/a\u003e\u003c/h1\u003e\n\u003cp\u003e\u003ca href=\"https://metabolomics.ucdavis.edu/core-services/rates-and-costs-for-services\"\u003ehttps://metabolomics.ucdavis.edu/core-services/rates-and-costs-for-services\u003c/a\u003e\u003c/p\u003e\n\u003ch1 id=\"uthsc\"\u003eUTHSC\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#uthsc\"\u003e\u003c/a\u003e\u003c/h1\u003e\n\u003cp\u003e\u003ca href=\"https://www.uthsc.edu/research/institutional-cores/pmc/documents/fy20-pmc-mpms-price-list.pdf\"\u003ehttps://www.uthsc.edu/research/institutional-cores/pmc/documents/fy20-pmc-mpms-price-list.pdf\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003eTargeted metabolite analyses:\n$107 / sample   \u003c/p\u003e\n\u003cp\u003eIdentification of Unknown Metabolites\n~ $ 5,000 / project  \u003c/p\u003e\n\u003ch1 id=\"duke\"\u003eDuke\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#duke\"\u003e\u003c/a\u003e\u003c/h1\u003e\n\u003cp\u003e\u003ca href=\"https://genome.duke.edu/cores-and-services/proteomics-and-metabolomics/pricing\"\u003ehttps://genome.duke.edu/cores-and-services/proteomics-and-metabolomics/pricing\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003eSample Cleanup Procedures (cost per method)\t$48\t$77\nSolid Phase Extraction (C18, HLB, MCX)\t\t\u003c/p\u003e\n\u003cp\u003eQuantitative Analysis (Metabolomics)\u003c/p\u003e\n\u003cp\u003eAll prices are on a per-sample basis.\u003c/p\u003e\n\u003cp\u003eDuke\tNon-Duke\nAbsoluteIDQ p180 Panel (kit from Biocrates, Inc.)\u003cem\u003e\t$52\t$84\nAbsolute IDQ p400 HR Panel (kit from Biocrates, Inc.)\u003c/em\u003e\t$72\t$116\nMxP Quant 500 Panel (kit from Biocrates, Inc.)\u003cem\u003e\t$84\t$135\nBile Acids Panel (kit from Biocrates, Inc.)\u003c/em\u003e\t$41\t$66\nOxylipins/Eicosanoid Panel\u003cstrong\u003e\t$145\t$233\nEnergy Metabolite Panel\t$64\t$103\nMethionine Pathway Panel\u003c/strong\u003e\t$88\t$142\nHydroxycholesterols Panel\u003cstrong\u003e\t$130\t$209\n27-Hydroxycholesterol quantitation\u003c/strong\u003e\t$76\t$122\nFatty Acids Panel\t$74\t$119\nFast HR Metabolite Profiling (RPLC OR HILIC, ESI+ OR ESI-)\t$142\t$229\nHigh Resolution Polar Metabolite Profiling, ESI+\t$207\t$333\nHigh Resolution Lipid Profiling, ESI+ and ESI-\t$252\t$406\nPK Analysis - method development\tContact\tContact\nper sample with in-house developed method\u003cstrong\u003e\t$29\t$47\nPolyamine Panel\u003c/strong\u003e\t$57\t$92\nShort Chain Fatty Acids Analysis\u003cstrong\u003e\t$61\t$98\nSphingosine S1P Panel\u003c/strong\u003e\t$44\t$71\nZipChip Profiling Method\t$66\t$106\u003c/p\u003e\n\u003cp\u003eData Analysis\u003c/p\u003e\n\u003cp\u003eDuke\tNon-Duke\nCustom Database Setup for MASCOT or Identify\t$25\t$40\nAdditional Database Searches\t$32\t$52\nConsulting and Extensive Data Mining (per hour)\t$64\t$103\u003c/p\u003e\n\u003ch1 id=\"metabo-centre-ca\"\u003eMetabo centre ca\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#metabo-centre-ca\"\u003e\u003c/a\u003e\u003c/h1\u003e\n\u003cp\u003e\u003ca href=\"https://www.metabolomicscentre.ca/services\"\u003ehttps://www.metabolomicscentre.ca/services\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003eUntargeted Metabolomics by RPLC-QTOF-MS and MS/MS\u003c/p\u003e\n\u003cblockquote\u003e\n\u003cp\u003e5000 features\nAcademic/Government:\u003c/p\u003e\n\u003c/blockquote\u003e\n\u003cp\u003e$140 for 2 RPLC-MS runs per sample (relative quantification);\n$280 for 2 RPLC-MS/MS runs per study (metabolite identification)\n$75/hour for data analysis\nIndustry:\u003c/p\u003e\n\u003cp\u003e$336 per sample (relative quantification)\n$672 per study (metabolite identification)\n$180/hour for data analysis\n20-200 µL or 10-50 mg lyophilized powder\nUntargeted metabolomics by GC×GC-TOFMS (EI only). Sample introduction options include liquid, SPME, direct headspace, thermal desorption\u003c/p\u003e\n\u003cblockquote\u003e\n\u003cp\u003e2000\nAcademic/Government:\u003c/p\u003e\n\u003c/blockquote\u003e\n\u003cp\u003e$200 + sample prep\n$75/hour for data analysis\nIndustry:\u003c/p\u003e\n\u003cp\u003e-$400 + sample preparation assuming standard GC×GC column configuration works – i.e. no custom method development - $180/hour for data analysis\u003c/p\u003e\n\u003cp\u003eUntargeted metabolomics by UPLC-FT-MS\n6000 - 10000\nAcademic/Government:\u003c/p\u003e\n\u003cp\u003e$400 (1st sample)\n$120 for 2 UPLC-MS runs per sample\n$240 for 4 UPLC-MS runs per sample\n$75/hour for data analysis\nIndustry:\u003c/p\u003e\n\u003cp\u003e$800 (1st sample)\n$240 for 2 UPLC-MS runs per sample\n$480 for 4 UPLC-MS runs per sample\n$150/hour for data analysis\n75 µL\u003c/p\u003e\n\u003ch1 id=\"colostate\"\u003eColostate\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#colostate\"\u003e\u003c/a\u003e\u003c/h1\u003e\n\u003cp\u003e\u003ca href=\"https://www.research.colostate.edu/bio/metabolomics-pricing/\"\u003ehttps://www.research.colostate.edu/bio/metabolomics-pricing/\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003eCSU\tNon-CSU Academic or Non-profit\tNon-CSU Commercial\nLC-Q-TOF-MS: Set-up Fee (charged regardless of sample #)\t$147\t$221\t$368\nLC-Q-TOF-MS: Instrument usage\t$74/hour\t$110/hour\t$184/hour\nLC-Q-TOF-MS: Sample Preparation\t$74/hour\t$110/hour\t$184/hour\nGC-MS: Set-up Fee (charged regardless of sample #)\t$114\t$172\t$286\nGC-MS Analysis – Instrument usage\t$41/hour\t$61/hour\t$102/hour\nGC-MS: Sample Preparation\t$74/hour\t$110/hour\t$184/hour\nGC-MS: Sample Derivatization\t$13/sample\t$19/sample\t$32/sample\nLC-QQQ-MS: Set-up Fee (charged regardless of sample #)\t$155\t$233\t$389\nLC-QQQ-MS: Instrument usage\t$82/hour\t$123/hour\t$205/hour\nLC-QQQ-MS: Assay development\t$622\t$932\t$1,554\nLC-QQQ-MS: Sample Preparation\t$74/hour\t$110/hour\t$184/hour\nGeneral Data Analysis\u003cem\u003e\t$74/hour\t$110/hour\t$184/hour\nData analysis or sample preparation training\u003c/em\u003e\t$134/hour\t$202/hour\t$336/hour\nNon-targeted Metabolite Profiling data analysis – tier 1\u003cstrong\u003e\t$74/project\t$110/project\t$184/project\nNon-targeted Metabolite Profiling data analysis – tier 2\u003c/strong\u003e\t$294/project\t$441/project\t$735/project\nNon-targeted Metabolite Profiling data analysis – tier 3**\t$1176/project\t$1764/project\t$2940/project\nPricing as of February 01, 2020. Prices are subject to change at any time.\u003c/p\u003e\n\u003cul\u003e\n\u003cli\u003eOne hour minimum\u003c/li\u003e\n\u003c/ul\u003e\n\u003cp\u003e**Tier 1 (raw data files converted to .cdf format, sample key), Tier 2 (raw data files converted to cdf, XCMS based peak detection and alignment with output data in csv format, outlier detection, PCA, simple ANOVA p-values, figures of output) , Tier 3 (raw data files converted to cdf, XCMS based peak detection and alignment with output data in csv format, outlier detection, PCA, simple ANOVA p-values, figures of output, metabolite annotation).\u003c/p\u003e\n\u003cp\u003ePlease note that costs are independent of result, and fees must still be charged for unsuccessful analysis in order to cover analysis time, instrument maintenance and consumables.\u003c/p\u003e\n\u003ch1 id=\"griffin-group\"\u003eGriffin group\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#griffin-group\"\u003e\u003c/a\u003e\u003c/h1\u003e\n\u003cp\u003e\u003ca href=\"https://www.metabolomics.bioc.cam.ac.uk/Facility\"\u003ehttps://www.metabolomics.bioc.cam.ac.uk/Facility\u003c/a\u003e\u003c/p\u003e\n\u003col start=\"2\"\u003e\n\u003cli\u003e\n\u003cp\u003eGC-FID analysis of total fatty acids. Fatty acids will be definitively identified by retention time matching with known standards and where necessary the acquisition of mass spectra using GC-MS and matching with compounds in the NIST library. Consumable cost: £24.00 per sample.\u003c/p\u003e\n\u003c/li\u003e\n\u003cli\u003e\n\u003cp\u003eGC-MS analysis of the aqueous extract of tissues, cells or biofluids using a MSTFA derivatisation procedure as detailed in Atherton et al., 2006. Metabolites will be identified by mass spectral matching with the NIST library and other publically available resources e.g. the Fiehn lab database. Consumable cost: £27.00 per sample.\u003c/p\u003e\n\u003c/li\u003e\n\u003cli\u003e\n\u003cp\u003eUPLC LC-MS analysis of lipids from tissue and serum extracts to identify bulk lipid changes as detailed in Roberts et al., 2008. This assay will use a UPLC QTOF Ultima and allows the analysis of intact lipids directly. Lipid identification will be carried out by MS/MS fragmentation and reference to libraries such as LipidMaps where possible. Consumable costs: £36.00 per sample.\u003c/p\u003e\n\u003c/li\u003e\n\u003c/ol\u003e\n\u003ch1 id=\"mac-anderson\"\u003eMac Anderson\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#mac-anderson\"\u003e\u003c/a\u003e\u003c/h1\u003e\n\u003cp\u003e\u003ca href=\"https://www.mdanderson.org/research/research-resources/core-facilities/metabolomics-facility/services.html\"\u003ehttps://www.mdanderson.org/research/research-resources/core-facilities/metabolomics-facility/services.html\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003eNon-targeted Metabolomics:\nProfiling (complete)\nQuantitative analysis of metabolites in non-targeted fashion from multiple fractions\nMultiple configurations available\nPricing:\nInternal \u0026#x26; Academic (In Network)-$600/sample\nAcademic (Out of Network)-$960 (+60% Overhead)\nNon-academic-$1200/sample (+60% Overhead)\u003c/p\u003e\n\u003ch1 id=\"metabo-iigb\"\u003eMetabo iigb\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#metabo-iigb\"\u003e\u003c/a\u003e\u003c/h1\u003e\n\u003cp\u003e\u003ca href=\"https://metabolomics.iigb.ucr.edu/services-and-pricing/\"\u003ehttps://metabolomics.iigb.ucr.edu/services-and-pricing/\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003eMost services start at around $74/sample, but the cost is largely dependent on sample preparation complexity and sample set size. Please contact the Metabolomics Core Team (\u003ca href=\"/mapp-dws-public/mailto:metabolomics@ucr.edu\"\u003emetabolomics@ucr.edu\u003c/a\u003e) to discuss options and pricing for your project.\u003c/p\u003e\n\u003chr\u003e\n\u003cstrong\u003eChildren\u003c/strong\u003e\n\u003col\u003e\n\u003cli\u003e\u003ca href=\"/mapp-dws-public/notes/YSBdEmydk8XLo8ssSQec2\"\u003eEval\u003c/a\u003e\u003c/li\u003e\n\u003cli\u003e\u003ca href=\"/mapp-dws-public/notes/fUD4wPYZe4AE8Gslj4qoA\"\u003ePh\u003c/a\u003e\u003c/li\u003e\n\u003cli\u003e\u003ca href=\"/mapp-dws-public/notes/yiNaYRHMNL83IomXDrDSG\"\u003eShiny\u003c/a\u003e\u003c/li\u003e\n\u003cli\u003e\u003ca href=\"/mapp-dws-public/notes/AjS4TQTF1i254OvXs20UW\"\u003eTable\u003c/a\u003e\u003c/li\u003e\n\u003c/ol\u003e","noteIndex":{"id":"nKP9hb3jEQKnhdgkHa5l0","title":"Root","desc":"","updated":1637350297192,"created":1630132422480,"custom":{"nav_order":0,"permalink":"/"},"fname":"root","type":"note","vault":{"fsPath":"vault"},"contentHash":"9dac15d3af8ef43be4ee8d4bca5e2202","links":[{"type":"ref","from":{"fname":"root","id":"nKP9hb3jEQKnhdgkHa5l0","vaultName":"vault"},"value":"projects.labelled-sugars.isotopic-ratio-calc-comp","position":{"start":{"line":7,"column":1,"offset":237},"end":{"line":7,"column":55,"offset":291},"indent":[]},"xvault":false,"to":{"fname":"projects.labelled-sugars.isotopic-ratio-calc-comp"}},{"type":"ref","from":{"fname":"root","id":"nKP9hb3jEQKnhdgkHa5l0","vaultName":"vault"},"value":"projects.labelled-sugars.isotopic-ratio-calc-comp","position":{"start":{"line":9,"column":1,"offset":293},"end":{"line":9,"column":110,"offset":402},"indent":[]},"xvault":true,"to":{"fname":"projects.labelled-sugars.isotopic-ratio-calc-comp","vaultName":"didier-reinhardt-group-IRayp1ECkrty3SxhP1Cap"}}],"anchors":{"welcome-to-dendron":{"type":"header","text":"Welcome to Dendron","value":"welcome-to-dendron","line":7,"column":0,"depth":1}},"children":["nj2zdhrspr8p4l15wb8dpg7","TRvMUEW7HVT7Vvny0AUF1","zbltzhf49jneruzhejqlzty","4ade8jh5lae5a1y61r2d7y0","vq7xwpdxi15poruey3ib0sh","0ycy0n1wprvx7h82emg4zgc","mrslbnp9bp5tia1ne4nnszu","RBxrtKZIWsMnHMVAyLlmc","mBtD1qPsLMRYntiuhCOBG","48173l5udd340kw1a4v9tuv","yohkugdcff8q2insh88fxpl","qb2q4vx5931adijds95fdbt","hva29wlmgjgu3vn9ghmpdii","huy4u2abu7my4w66cjzh04u","zg9lbox3rbxl0qvkctyv8qm","wgfph36zgefqywvcq5d3ji0"],"parent":null,"data":{},"body":"# Welcome to Dendron\n\nThis is the root of your dendron vault. If you decide to publish your entire vault, this will be your landing page. You are free to customize any part of this page except the frontmatter on top. \n\nCroosvaults test\n\n![[projects.labelled-sugars.isotopic-ratio-calc-comp]]\n\n![[dendron://didier-reinhardt-group-IRayp1ECkrty3SxhP1Cap/projects.labelled-sugars.isotopic-ratio-calc-comp]]\n\n"},"collectionChildren":null,"customHeadContent":null,"config":{"version":5,"dev":{"enablePreviewV2":true},"commands":{"lookup":{"note":{"selectionMode":"extract","confirmVaultOnCreate":false,"leaveTrace":false,"bubbleUpCreateNew":true,"vaultSelectionModeOnCreate":"smart","fuzzThreshold":0.2}},"insertNote":{"initialValue":"templates"},"insertNoteLink":{"aliasMode":"none","enableMultiSelect":false},"insertNoteIndex":{"enableMarker":false},"randomNote":{},"copyNoteLink":{"aliasMode":"title"},"templateHierarchy":"template"},"workspace":{"vaults":[{"fsPath":"vault"}],"journal":{"dailyDomain":"daily","name":"journal","dateFormat":"y.MM.dd","addBehavior":"childOfDomain"},"scratch":{"name":"scratch","dateFormat":"y.MM.dd.HHmmss","addBehavior":"asOwnDomain"},"graph":{"zoomSpeed":1,"createStub":false},"enableAutoCreateOnDefinition":false,"enableXVaultWikiLink":false,"enableRemoteVaultInit":true,"workspaceVaultSyncMode":"noCommit","enableAutoFoldFrontmatter":false,"maxPreviewsCached":10,"maxNoteLength":204800,"task":{"name":"","dateFormat":"","addBehavior":"childOfCurrent","statusSymbols":{"":" ","wip":"w","done":"x","assigned":"a","moved":"m","blocked":"b","delegated":"l","dropped":"d","pending":"y"},"prioritySymbols":{"H":"high","M":"medium","L":"low"},"todoIntegration":false,"createTaskSelectionType":"selection2link","taskCompleteStatus":["done","x"]},"enableUserTags":true,"enableHashTags":true,"dendronVersion":"0.71.0","enableEditorDecorations":true,"enableFullHierarchyNoteTitle":false},"preview":{"enableFMTitle":true,"enableNoteTitleForLink":true,"enablePrettyRefs":true,"enableKatex":true,"enableFrontmatterTags":true,"enableHashesForFMTags":false,"automaticallyShowPreview":false},"publishing":{"enableFMTitle":true,"enableNoteTitleForLink":true,"enablePrettyRefs":true,"enableKatex":true,"copyAssets":true,"siteHierarchies":["root"],"writeStubs":false,"siteRootDir":"docs","seo":{"title":"MAPP Metabolomics Public Dendron","description":"Personal knowledge space"},"github":{"enableEditLink":true,"editLinkText":"Click here to edit this page on Github !","editBranch":"main","editViewMode":"edit","editRepository":"https://github.com/mapp-metabolomics-unit/mapp-dws-public"},"enableSiteLastModified":true,"enableFrontmatterTags":true,"enableHashesForFMTags":false,"enableRandomlyColoredTags":true,"enableTaskNotes":true,"enablePrettyLinks":true,"searchMode":"lookup","assetsPrefix":"/mapp-dws-public","siteUrl":"https://mapp-metabolomics-unit.github.io","duplicateNoteBehavior":{"action":"useVault","payload":["vault"]},"siteFaviconPath":"favicon.ico","siteIndex":"root"}}},"__N_SSG":true},"page":"/notes/[id]","query":{"id":"RBxrtKZIWsMnHMVAyLlmc"},"buildId":"HcHSndH05GsIFs4lOyTZF","assetPrefix":"/mapp-dws-public","isFallback":false,"gsp":true,"scriptLoader":[]}</script></body></html>